target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -204,17 +204,21 @@ hauppauge,broadway)
 	set_wifi_led "$boardname:red:wps_active"
 	;;
 hiwifi,hc5661)
-	ucidef_set_led_netdev "internet" "internet" "$boardname:blue:internet" "eth0.2"
+	ucidef_set_led_switch "internet" "internet" "$boardname:blue:internet" "switch0" "0x1"
```

Comment:
Maybe use 0x01 instead of 0x1 here and below for consistency with other definitions...